app/helpers/admin/users_helper.rb

```diff
@@ -89,5 +89,25 @@ def organization_tooltip(organization_names, count, imposed_limit: 0)
         str + " & #{overflow_count(count, imposed_limit: imposed_limit)} others"
       end
     end
+
+    # Returns a boolean that indicates whether a user can be banished
+    # A user can be banished when:
+    # - the users account was created in the period before the last 100 days ago
+    # and there are no comments from the user in the period before the last 100 days.
+    # - if the logged in user is a super admin or support admin
+
+    # Some practical scenarios:
```

Comment:
I struggled to understand this a little and so I added a practical scenario. Happy to take it out if folks think it's unnecessary.